fds/services/dvc_service.py

```diff
@@ -209,6 +209,7 @@ def __add(self, add_argument: str):
         return "DVC add successfully executed"
 
     def add(self, add_argument: str) -> Any:
+        self.printer.warn("Adding...")
```

Comment:
I think this print should be in the `fds_service` where this call is made

Reply:
done